app/ancillaryFactory/src/appRegistry/AncillaryFactory.js

```diff
@@ -0,0 +1,50 @@
+// @flow strict
+
+import * as React from 'react';
+import { AuthContext } from '@kiwicom/mobile-relay';
+
+import {
+  type AncillaryDefinition,
+  ancillariesDefinitions,
+} from '../config/ancillariesDefinitions';
+import NoAncillaryMessage from '../components/NoAncillaryMessage';
+
+type Props = {|
+  +service: string,
+  +bookingId: number,
+  +kwAuthToken: string,
+|};
+
+class AncillaryFactory extends React.Component<Props> {
+  getRequestedAncillary() {
+    const { service } = this.props;
+
+    const ancillary: AncillaryDefinition =
+      ancillariesDefinitions[service.toUpperCase()];
+
+    if (ancillary && typeof ancillary.renderComponent === 'function') {
+      return ancillary.renderComponent;
+    }
```

Comment:
you don't need this if else `return ancillary?.renderComponent;` is doing the exact same thing